sdk/resourcemanager/azure-resourcemanager-containerinstance/src/main/java/com/azure/resourcemanager/containerinstance/implementation/ContainerGroupImpl.java

```diff
@@ -640,6 +649,15 @@ public DnsConfiguration dnsConfig() {
         return this.innerModel().dnsConfig();
     }
 
+    @Override
+    public DnsNameLabelReusePolicy autoGeneratedDomainNameLabelScope() {
+        if (this.innerModel().ipAddress() != null && this.innerModel().ipAddress().dnsNameLabel() != null) {
```

Comment:
Just check `this.innerModel().ipAddress() != null`.

Give user what service gives SDK. No need to do extra check on "maybe related" property.